Makefile

```diff
@@ -8,7 +8,7 @@ QEMU ?= qemu-system-x86_64 -enable-kvm -cpu host
 iso: $(BASENAME).iso
 
 $(BASENAME).iso:
-	lb config
+	lb config --archive-areas "main contrib non-free non-free-firmware"
```

Comment:
Is this change required? I've already made a similar one by adding `non-free-firmware` to `auto/config` here: https://github.com/aursulis/liox/blob/master/auto/config#L26

Reply:
For some reason, I still had issues while building with the Makefile. Possibly differences between distros? I'm 90% sure this script was being run, as I saw the LIOX_ARCH error message when I forgot to set variable. I will look into this tomorrow

Reply:
Tested again, still fails.
![image](https://github.com/aursulis/liox/assets/16293470/df3afddc-86e1-420d-9950-744e5e784625)